tests/foreman/installer/test_installer.py

```diff
@@ -1586,3 +1586,50 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.destructive
```

Comment:
You can now get a satellite deployed on-demand for the scope of your test, and it won't require a destructive marker.

Please see #8742 for examples